lib/rsgem/tasks/set_license_file.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module RSGem
+  module Tasks
+    class SetLicenseFile < Base
+      def perform
+        license[2] = 'Copyright (c) 2020 Rootstrap'
```

Comment:
Given that we don't know if future versions of bundler will change the LICENSE.txt file they generate, what do you think of updating this using regex? (as in the other tasks that replace content)